package.json

```diff
@@ -35,12 +35,13 @@
     "doc": "./docs"
   },
   "dependencies": {
-    "appium-android-driver": "^1.10.19",
-    "appium-base-driver": "^2.0.8",
+    "appium-android-driver": "^1.10.27",
+    "appium-base-driver": "^2.0.17",
     "appium-fake-driver": "^0.1.10",
     "appium-ios-driver": "^1.12.25",
     "appium-logger": "^2.1.0",
     "appium-selendroid-driver": "^1.3.4",
+    "appium-uiautomator2-driver":"^0.0.1-beta1",
```

Comment:
I'm not sure this is the best way to do this. `^0.0.1-beta1` will only _ever_ match `0.0.1-beta1`, so this package would need to be changed in order to get any published changes in the driver. 

It would be much better to just publish 0.0.x versions of the driver (using the `beta` tag if you want (https://docs.npmjs.com/getting-started/using-tags#publishing-with-tags)) and when everything is stable, publish `1.0.0`. This way, until the first version the semver `^0.0.1` would catch each iteration.